Makefile

```diff
@@ -14,8 +14,5 @@ integration:
 validate:
 	mvn checkstyle:check@checkstyle-execution dependency:analyze@analyze
 
-codecov:
-    mvn cobertura:cobertura
```

Comment:
there were spaces here not tabs, so this make target has been failing on travis ci since it was introduced